retrieval/discovery/consul.go

```diff
@@ -95,6 +96,17 @@ func NewConsulDiscovery(conf *config.ConsulSDConfig) *ConsulDiscovery {
 		scrapedServices: map[string]struct{}{},
 		services:        map[string]*consulService{},
 	}
+	// If the datacenter isn't set in the clientConf, let's get it from the local Consul agent
+	// (Consul default is to use local node's datacenter if one isn't given for a query)
```

Comment:
@fabxc Do you want me to remove "// (Consul's default is to use the local node's datacenter if none is given for a query.)"?